README.md

```diff
@@ -1,68 +1,212 @@
 # veritable-ui
 
-UI for `Veritable` utilizing [TSOA](https://tsoa-community.github.io/docs/getting-started.html), [HTMX](https://htmx.org/) and [@kitajs/html](https://www.npmjs.com/package/@kitajs/html) for JSX templating.
+A user interface for `Veritable` that allows to manage connections across supply chain and perform queries in relation to the supply chain. Utilizing [TSOA](https://tsoa-community.github.io/docs/getting-started.html), [HTMX](https://htmx.org/) and [@kitajs/html](https://www.npmjs.com/package/@kitajs/html) for JSX templates.
 
-## Setup
+#### Table of Contents
 
-To install dependencies:
+- [Setup](#setup)
+- [Getting Started](#getting-started)
+- [Development Mode/Local dev](#development-mode)
+- [Environment Variables](#environment-variables)
+- [Testing](#testing)
 
-```bash
+## Setup
+veritable-ui depends on a few external services:
+- [veritable-cloudagent](https://github.com/digicatapult/veritable-cloudagent) - APIs for managing connections, credentials and messages.
+- `keycloak` - runs as a docker image for each node that handles authentication and users.
+- `docker` - for orchestrating multiple nodes and running other dependencies such as database (PostgreSQL) along with node specific environment variables
+
+### Prerequisites
+> :warning: last updated: 01/08/2024
+- docker v27.0.3+
+- npm 10.0.0+
+- node 22.0.0+
+- keycloak:25.0.2+ - included in the docker-compose.yml
+- veritable-cloudagent@latest - last used -> v0.9.42
```

Comment:
these aren't prerequisites as they will be automatically managed in the docker compose as you've said. I think including them is confusing so remove?